firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/DeferredExtensions.kt

```diff
@@ -0,0 +1,17 @@
+package dev.gitlive.firebase.firestore
+
+import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.Deferred
+
+@PublishedApi
+internal fun <T, R> Deferred<T>.convert(converter: (T) -> R): Deferred<R> {
```

Comment:
is there a better way to convert a result of a deferred?

Reply:
Can't think of anything.